examples/handling/georeferencing/crop_raster.py

```diff
@@ -0,0 +1,52 @@
+"""
+Crop a raster
+=============
+
+This example demonstrates the cropping of a raster using :func:`geoutils.Raster.crop`.
+"""
+# %%
+# We open a raster and vector, and subset the latter.
+
+# sphinx_gallery_thumbnail_number = 2
+import geoutils as gu
+
+rast = gu.Raster(gu.examples.get_path("everest_landsat_b4"))
+vect = gu.Vector(gu.examples.get_path("everest_rgi_outlines"))
+vect = gu.Vector(vect.ds[vect.ds["RGIId"] == "RGI60-15.10055"])
+
+# %%
+# The first raster has larger extent and higher resolution than the second one.
```

Comment:
The raster has a larger extent than the vector

Reply:
Corrected